abdm_python_integrator/abha/tests/test_abha_creation.py

```diff
@@ -0,0 +1,89 @@
+from unittest.mock import patch
+
+from django.contrib.auth.models import User
+from rest_framework import status
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+
+
+class TestABHACreation(APITestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.invalid_req_msg = "Unable to process the current request due to incorrect data entered."
+        cls.user = User.objects.create_superuser(username='test_user', password='test')
+
+    def setUp(self) -> None:
+        self.client.force_authenticate(user=self.user)
+
+    @staticmethod
+    def _mock_abdm_http_post(url, payload):
+        abdm_txn_id_mock = {"txnId": "1234"}
+        return {
+            "v1/registration/aadhaar/generateOtp": abdm_txn_id_mock,
```

Comment:
These should be coming from constants. For e.g.`const.GENERATE_AADHAAR_OTP_URL`

Reply:
Addressed in 277b9d17b5dfe5d0376cf986d8aee460b0322492